examples/convert_disk.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Show how to download disk snapshots using imageio client.
+Requires the ovirt-imageio-client package.
+"""
+
+import time
+
+from contextlib import closing
+
+import ovirtsdk4.types as types
+
+from helpers import common
+
+
+def parse_args():
+    parser = common.ArgumentParser(description="Download disk snapshot")
+
+    parser.add_argument("disk_id", help="The disk's id")
+
+    parser.add_argument(
+        "-f", "--format",
+        choices=("raw", "qcow2"),
+        help=("The format the disk will converted to (qcow2, raw"))
+
+    parser.add_argument(
+        "--sparse",
+        type=bool,
```

Comment:
this can't actually be a bool, if it's not passed we don't want to change the allocation policy